common/metrics/defs.go

```diff
@@ -1693,6 +1702,10 @@ var ScopeDefs = map[ServiceIdx]map[int]scopeDefinition{
 	Server: {
 		ServerTlsScope: {operation: "ServerTls"},
 	},
+	UnitTestService: {
+		TestScope1: {operation: "test_scope_1_operation"},
+		TestScope2: {operation: "test_scope_2_operation"},
+	},
```

Comment:
do we have to add those here for testing? Is there a way to put them in test package?

Reply:
We currently initialize clients using this global variable. I'll look to separate it in other PRs.